platform/features/conductor-redux/src/README.md

```diff
@@ -0,0 +1,187 @@
+## Notes
+API is notional for now, based on use-cases identified below. Possible the 
+use cases are not sufficient, so please include in comments 
+any other use cases you'd like to see. 
+
+Plan now is to start building out test suite for the use cases identified below
+in order to get the API functional. Need to discuss how UI aspects of timeline will be implemented.
+Propose in place refactoring of existing timeline rather than starting again.
+
+Some caveats / open questions
+* I don't understand the use case shown on page 52 of UI sketches. It shows RT/FT, with deltas, 
+with inner interval unlocked. Not sure what result would be, has inner end switched to fixed?
+Also example on page 55 in real-time where inner end < now. Is there a use case for this? Semantically, it's saying
+show me real time, but stale data. Why would a user want this? My feeling is that if the inner 
+OR outer ends are moved behind NOW in real-time mode then you drop into historical mode.
+* For the API itself, have ignored question of how it's namespaced / exposed. 
+Examples assume global namespace and availability from window object. 
+For now API implemented as standard standard Require JS AMDs. Could attach 
+to window from bundle.js. Perhaps attaching to window not best approach though...
+* Have not included validation (eg. start time < end time) or any other 
+business logic such as what happens when outer interval gets dragged 
+within range of inner interval. Focus is on teasing out the public API 
+right now. 
+* Time systems are vague right now also, I don't know how they're going 
+to work or whether any API has yet been specified.
+* Not clear on the differences between real-time and follow-time as it 
+concerns the time conductor? For now the API has an end bounds mode
+of FOLLOW which automatically tracks current time, and a start time mode 
+of RELATIVE. I can envision a real-time plot that is not in follow time mode, 
+but not sure what implication is for time conductor itself and how it 
+differs from an historical plot?
+* Should the time conductor be responsible for choosing time system / domain? Currently 
+it is.
+
+## Use Cases
+1. Historical session is loaded and needs to set bounds on time conductor
+2. User changes time of interest
+3. Plot controller listens for change to TOI
+4. Real-time object is loaded and time conductor is set up in real-time mode showing last 15 minutes of data.
+5. Switch to follow time. Set inner end to now, and outer end to now + 1 hour
+6. Plot controller updated on tick
+7. Plot Controller updated when user changes bounds
+8. Conductor controller needs to update bounds and mode on TC when user changes bounds
+
+### Additional possible use-cases
+1. Telemetry adapter wants to indicate presence of data at a particular time
+2. Time conductor controller wants to paint map of data availability.
```

Comment:
Agree with omitting these use cases. This information is useful visually but not relevant to the information the time conductor is modeling (user's desired time frame and settings.)